modules/cart/tests/src/Functional/CartBlockTest.php

```diff
@@ -41,4 +41,17 @@ public function testCountText() {
     $this->assertSession()->pageTextNotContains('3 items');
   }
 
+  /**
+   * Tests the order items display.
+   */
+  public function testOrderItemsDisplay() {
+    $this->drupalGet('<front>');
```

Comment:
How is this test method different from the one above? They both add an entity to cart and assert the count. Which makes me think the previous one should have caught the bug?